plugin.video.filmarkivet/default.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
```

Comment:
It'd be nice if at some point you can move most of this code to a module. As the addon entrypoint, this is the only file that's not compiled to bytecode by Kodi's python interpreter. You'll have a bit of performance increase if you just import the code and run:

```python
from resources.lib import yourcode

yourcode.run()
```